s3transfer/manager.py

```diff
@@ -45,7 +47,8 @@ def __init__(self,
                  max_submission_queue_size=1000,
                  max_io_queue_size=1000,
                  num_download_attempts=5,
-                 max_in_memory_upload_chunks=10):
+                 max_in_memory_upload_chunks=10,
+                 max_in_memory_download_chunks=10):
```

Comment:
Would customers want these to be different? If you're limiting chunks in memory, does it matter what put it there?